xExtension-ImageProxy/i18n/en/ext.php

```diff
@@ -5,7 +5,7 @@
 		'proxy_url' => 'Proxy URL',
 		'scheme_http' => 'Proxy HTTP',
 		'scheme_https' => 'Proxy HTTPS',
-		'scheme_default' => 'Proxy unspecified',
+		'scheme_default' => 'Proxy unspecified scheme',
```

Comment:
I think I'll make it URL scheme. That's clearer. Pity the visuals are a bit worse.
![image](https://github.com/user-attachments/assets/d667db83-dfe5-43ea-a01d-772069afe4bd)